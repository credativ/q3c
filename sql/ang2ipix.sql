select q3c_ang2ipix(0, 0); 
select q3c_ang2ipix(0, 91); 
select q3c_ang2ipix(0, 90); 
select q3c_ang2ipix(-1, 0); 
select q3c_ang2ipix(359, 0); 
select q3c_ang2ipix('nan'::double precision,0);
select q3c_dist(0,0,1,0);
select q3c_dist(0,0,0,1);
select q3c_dist(0,0,90,0);
select q3c_dist(0,0,0,90);
select sum((q3c_ang2ipix(((q3c_ipix2ang(q3c_ang2ipix(ra,dec)))[1]),(q3c_ipix2ang(q3c_ang2ipix(ra,dec)))[2])=q3c_ang2ipix(ra,dec))::int) from test;
select sum((q3c_ang2ipix(ra::real,dec::real) = q3c_ang2ipix( (ra::real)::double precision, (dec::real)::double precision))::int) from test;
