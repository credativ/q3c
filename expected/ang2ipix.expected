    q3c_ang2ipix     
---------------------
 2017612633061982208
(1 row)

    q3c_ang2ipix    
--------------------
 480383960252852906
(1 row)

    q3c_ang2ipix    
--------------------
 480383960252852906
(1 row)

    q3c_ang2ipix     
---------------------
 1825388318008017157
(1 row)

    q3c_ang2ipix     
---------------------
 1825388318008017157
(1 row)

 q3c_ang2ipix 
--------------
             
(1 row)

 q3c_dist 
----------
        1
(1 row)

 q3c_dist 
----------
        1
(1 row)

 q3c_dist 
----------
       90
(1 row)

 q3c_dist 
----------
       90
(1 row)

   sum   
---------
 1000000
(1 row)

   sum   
---------
 1000000
(1 row)

